pkg/auth/signup.go

```diff
@@ -68,7 +67,7 @@ func Signup(pg *db.DB, logger *zap.SugaredLogger) echo.HandlerFunc {
 			Password: hash,
 			Name:     params.Name,
 			Email:    params.Email,
-			ImageID:  common.DefaultImageUuid,
+			ImageID:  "00000000-0000-0000-0000-000000000000",
```

Comment:
if user image is no longer a required field, you can modify the query to not include this

Reply:
you should also remove this in `user.sql`